helm-values/traction/values-sandbox.yaml

```diff
@@ -0,0 +1,85 @@
+ingressSuffix: .apps.silver.devops.gov.bc.ca
+acapy:
+  image:
+    pullPolicy: Always
+  secret:
+    adminApiKey:
+      generated: true
+    pluginInnkeeper:
+      generated: true
+  plugin-config.yml:
+    traction_innkeeper:
+      innkeeper_wallet:
+        print_key: true
+        print_token: true
+        connect_to_endorser:
+          - endorser_alias: endorser
+            ledger_id: bcovrin-test
+        create_public_did:
+          - bcovrin-test
+      reservation:
+        expiry_minutes: 2880
+        auto_approve:  true
+        auto_issuer: true
+      autoscaling:
+        enabled: false
+        minReplicas: 1
+        maxReplicas: 10
+        targetCPUUtilizationPercentage: 80
+        targetMemoryUtilizationPercentage: 80
+        stabilizationWindowSeconds: 300
+  resources:
+    limits:
+      cpu: 400m
+      memory: 1600Mi
+    requests:
+      cpu: 200m
+      memory: 820Mi
+  networkPolicy:
+    enabled: true
+    ingress:
+      enabled: true
+      namespaceSelector:
+        network.openshift.io/policy-group: ingress
+tenant_proxy:
+  image:
+    pullPolicy: Always
+  networkPolicy:
+    enabled: true
+    ingress:
+      enabled: true
+      namespaceSelector:
+        network.openshift.io/policy-group: ingress
+ui:
+  image:
+    pullPolicy: Always
+  oidc:
+    active: false
+    showInnkeeperAdminLogin: false
+    showWritableComponents: false
```

Comment:
You forgot these settings, turned them to true in the previous file and not in this one.

Reply:
@esune - Thanks for the feedback, I have made changes accordingly and re-deployed the sandbox.